Portfolio/index.html

```diff
@@ -1,177 +1,205 @@
+            <img src="/johnDoe.png" alt="Css Img" width="300px" height="300px">
+            <h1><b>Front End Developer</b></h1>
+            <h3>Get To Know me</h3>
+            <p><b>I'm a seasoned front-end web developer with a passion for crafting exceptional digital experiences.
+                    With over 2 years of hands-on experience, i bring a deep proficiency in HTML, CSS, JavaScript,
+                    TypeScript, and a keen mastery of modern frameworks like React.js, React Native, Svelte.js. I
+                    specialize in building responsive, interactive, and visually appealing user interfaces.
+                    Additionally, my proficiency in Firebase ensures secure and efficient backend integration, making me
+                    a versatile and dynamic developer ready to tackle any project</b>
         </div>
-        <p>Hello there! I'm Omparth Maruti Chougale, a passionate Software Engineer with a keen interest in Web Development. I love to code and to play games.</p>
-    </section>
+        </p>
 
-    <section>
-        <h2>My Skills</h2>
-        <ul>
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-            <li>Python</li>
-            <li>Photoshop</li>
-        </ul>
+        <h4>
+            <div>Name:John Doe</div>
+            <div>Age:24</div>
+            <div>Experience: 2years</div>
+        </h4>
+        </div>
     </section>
 
-    <section class="Exper">
-        <h2>Experience</h2>
-        <p>I have over fresher of experience. Throughout my career, I've worked on projects related to web development.</p>
-    </section>
+    <!-- skills -->
+    <section class="skills" id="skills">
 
-    <section>
-        <h2>Education</h2>
-        <p>I graduated from KIT College of engineering, Kolhapur with a degree in Btech. Additionally, I've completed courses in computer and Science.</p>
-    </section>
+        <h1>SKILLS</h1>
 
-    <section>
-        <h2>Contact Me</h2>
-        <form action="mailto:omparthchougale2182002@gmail.com" method="GET">
-            <label for="subject">Subject:</label><br>
-            <input type="text" id="subject" name="subject"><br>
-            <label for="message">Message:</label><br>
-            <textarea id="message" name="body"></textarea><br>
-            <input type="submit" value="Send">
-        </form>
-    </section>
+        <h4> <img src="/html.icon.png" alt="Css Image" width="30px" height="30px">
+            <div class="skilltext">HTML</div>
+        </h4>
 
-    <footer>
-        <h2>Connect with Me</h2>
-        <p>Feel free to connect with me on:</p>
-        <ul>
-            <li><a href="https://www.linkedin.com/in/omparth-chougale-582b89245/" target="_blank">LinkedIn</a></li>
-            <li><a href="https://twitter.com/Omparth2" target="_blank">Twitter</a></li>
-            <li><a href="https://github.com/omparth" target="_blank">GitHub</a></li>
-        </ul>
-    </footer>

```

Comment:
The closing </b> tag at line 113 is misplaced and should be moved before the closing </p> tag at line 111.